### PR TITLE
Change default value in ToDecibels, add one test

### PR DESCRIPTION
**Pull request by szalpal**
Signed-off-by: Michał Szołucha <mszolucha@nvidia.com>

#### Why we need this PR?
After seeing in the `ToDecibels` documentation:
> reference (float, optional, default = 1.0)

I've spent half of Thursday on figuring out, why this doesn't give me good results. Turns out, that this is utterly not true: the default value in this case **is not** 1.0, it's max of signal instead.
Thus I propose to change the default value to 0.0. I know, that this value is incorrect and makes no sense, thus I'm introducing `ENFORCE` to secure runtime in this case.
Thanks to this change, user won't be mislead, that default value is 1.0, since it is not.

Additionally, I'm including a test that I wrote when "debugging" the problem.

### CODE REVIEW

**Comment by szalpal** [comment]
!build

File: dali/test/python/test_operator_to_decibels.py [hunk 1]
```diff
@@ -94,3 +94,60 @@ def test_operator_to_decibels_vs_python():
                                                             (20.0, 1e-6, -120.0, (2, 3, 40))]:
                 yield check_operator_to_decibels_vs_python, device, batch_size, shape, \
                     multiplier, reference, cutoff_db
+
+
+class NaturalLogarithmPipeline(Pipeline):
```
Comment (JanuszL) [inline dali/test/python/test_operator_to_decibels.py]: You can add a test for this enforce as well.

Comment (szalpal) [reply dali/test/python/test_operator_to_decibels.py]: Done

**Comment by szalpal** [comment]
!build

**Pull request merged**